addons/website_event/tests/test_website_event.py

```diff
@@ -58,6 +59,41 @@ def test_canonical_url(self):
 
 @tagged('post_install', '-at_install')
 class TestWebsiteAccess(HttpCaseWithUserDemo, TestWebsiteEventCommon):
+    @staticmethod
+    def are_names_in_order(names_in_expected_order, text):
+        last_index = -1
+        if isinstance(text, bytes):
+            text = text.decode('utf-8')
+        for name in names_in_expected_order:
+            current_index = text.find(name, last_index + 1)
+            if current_index == -1:
+                if text.find(name) != -1:
+                    return False
+                else:
+                    raise Exception(f"I can't check ordering when name: '{name}' is not present in the text.")
+            elif current_index <= last_index:
+                return False
+            last_index = current_index
+        return True
+
+    @freeze_time('2024-09-1')
+    def test_event_ordering(self):
+        all_events = self.env['event.event'].search([])
```

Comment:
Could have
```suggestion
        all_events = self.env['event.event'].search([], order='id desc')
```
and avoided sorting later entirely.

But would avoid doing a search as it may lead to the test relying on demo/non-test data
Instead I would remove this line and use the events created in the set-up with `self.events`